### PR TITLE
TS Clean-up - Move Classes and Rename Packages

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

As a part of the Test Suite clean up, classes has been moved to a packages named according to `io.quarkus.ts.MODULE_NAME.SCENARIO_NAME`. Exact rules how to translate `MODULE_NAME` to java package name has been relaxed in order to make the package names readable. That's compliant with the existing state that either merge module name to one worded package name, or join module names by dots. 

Also fixed tests `ChuckNorrisResourceIT#endpointShouldTrace` and `ChuckNorrisResourceIT#httpClientShouldHaveHisOwnSpan` as  Jagger service `vertx-web-client` is tracing operating called `/trace/ping`, however tests were looking for `trace/ping` operation and thus failed. Similarly, `io.quarkus.ts.http.graphql.telemetry.TelemetryIT#verifyTelemetry` operation name has been changed to `/graphql`, which make the test pass.

Keycloak version used in `DevModeKeycloakDevServiceUserExperienceIT` is upgraded to 18.0. as upstream docs says `In Keycloak 17 the default distribution is now powered by Quarkus, 
while the legacy WildFly powered distribution will still be around until
 June 2022 we highly recommend starting the migration as soon as 
possible.` and using Keycloak 13.0.1 caused test failures of `DevModeKeycloakDevServiceUserExperienceIT`.

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [x] Refactoring

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by michalvavrik** [comment]
cc @jsmrcka

**Review by pjgg — CHANGES_REQUESTED** [review]
Also could you double-check the following modules:
- http/jaxrs: `io.quarkus.ts.security.core` ? 
- http/http-static: `io.quarkus.ts.http` -> `io.quarkus.ts.http.static`
- http/reactive-routes: `io.quarkus.ts` -> `io.quarkus.ts.http.reactiveroutes`
- http/server-undertow: `io.quarkus.ts.filters` -> `io.quarkus.ts.http.undertow.filters` (aplies to all packages)
- http/vertx-web-client: `io.quarkus.qe.vertx` -> `io.quarkus.ts.http.vertx` (applies to all packages)
- infinispan-client: `io.quarkus.ts.infinispan` -> `io.quarkus.ts.http.infinispan`
...and so on (in general try to add the module name to the packages definition) 
`io.quarkus.ts.MODULE_NAME.SCENARIO_NAME`

File: websockets/quarkus-websockets/src/test/java/io/quarkus/ts/websockets/producer/WebSocketsProducerIT.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 import io.quarkus.test.services.QuarkusApplication;
 
 @QuarkusScenario
-public class ChatIT {
+public class WebSocketsProducerIT {
```
[the remark this answers is not in the document]
Comment (michalvavrik) [reply websockets/quarkus-websockets/src/test/java/io/quarkus/ts/websockets/producer/WebSocketsProducerIT.java]: Make sense, thank you

File: websockets/websockets-client/src/test/java/io/quarkus/websockets/client/WebSocketsClientIT.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package io.quarkus.ts;
+package io.quarkus.websockets.client;
```
[the remark this answers is not in the document]
Comment (michalvavrik) [reply websockets/websockets-client/src/test/java/io/quarkus/websockets/client/WebSocketsClientIT.java]: Absolutely, I think that was the intention and missing "ts" was my typo. Fixed. Thank you

**Comment by michalvavrik** [comment]
> * http/http-static: `io.quarkus.ts.http` -> `io.quarkus.ts.http.static`
That's not a valid java package name.

**Comment by michalvavrik** [comment]
@pjgg I have run a tests before and after every refactoring in order to determine whether I brought in any new failures. The failures can be linked to the failures in Daily runs and IMHO should be resolved separately (a big commit as this is asking for a merge hell).

**Comment by michalvavrik** [comment]
Please note I also updated PR description. There is other work on clean up that should be done in a separate PRs.

File: http/graphql-telemetry/src/test/java/io/quarkus/ts/http/graphql/telemetry/TelemetryIT.java [hunk 3]
```diff
@@ -32,8 +30,8 @@ public class TelemetryIT {
 
     @Test
     void verifyTelemetry() {
-        Response classic = sendQuery(createQuery("friend(name:\"Aristotle\"){name}"));
-        Response reactive = sendQuery(createQuery("friend_r(name:\"Aristotle\"){name}"));
+        Response classic = Utils.sendQuery(Utils.createQuery("friend(name:\"Aristotle\"){name}"));
```
Comment (pjgg) [inline http/graphql-telemetry/src/test/java/io/quarkus/ts/http/graphql/telemetry/TelemetryIT.java:33]: Could you move on the "Utils" static class name to an import as it's nowadays

File: http/jaxrs-reactive/src/test/java/io/quarkus/ts/http/jaxrs/reactive/HttpCachingResourceIT.java [hunk 4]
```diff
@@ -57,6 +58,6 @@ private ValidatableResponse whenGet(String path) {
                 .get(BASE_PATH + path)
                 .then()
                 .statusCode(HttpStatus.SC_OK)
-                .body(is(HttpCachingResource.RESPONSE));
+                .body(CoreMatchers.is(HttpCachingResource.RESPONSE));
```
Comment (pjgg) [inline http/jaxrs-reactive/src/test/java/io/quarkus/ts/http/jaxrs/reactive/HttpCachingResourceIT.java:61]: Moves "CoreMatchers" to an import

File: http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnRequestBodyRouteHandlerIT.java [hunk 5]
```diff
@@ -29,10 +26,11 @@ public void shouldGetValidationErrorWhenRequestFirstCodeIsWrong() {
                 .statusCode(HttpStatus.SC_BAD_REQUEST)
                 .extract().as(ValidationErrorResponse.class);
 
-        assertValidationErrorTitle(response);
-        assertValidationErrorDetails(response);
-        assertValidationErrorStatus(response, HttpStatus.SC_BAD_REQUEST);
-        assertValidationErrorField(response, "validateRequestBody.param.firstCode", "First code must have 3 characters");
+        ValidationAssertions.assertValidationErrorTitle(response);
```
Comment (pjgg) [inline http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnRequestBodyRouteHandlerIT.java:29]: "ValidationAssertions" -> import...

File: http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnRequestBodyRouteHandlerIT.java [hunk 6]
```diff
@@ -48,11 +46,13 @@ public void shouldGetValidationErrorsWhenFirstAndSecondCodesAreWrong() {
...
                 .statusCode(HttpStatus.SC_BAD_REQUEST)
                 .extract().as(ValidationErrorResponse.class);
-        assertValidationErrorTitle(response);
-        assertValidationErrorDetails(response);
-        assertValidationErrorStatus(response, HttpStatus.SC_BAD_REQUEST);
-        assertValidationErrorField(response, "validateRequestBody.param.firstCode", "First code must have 3 characters");
-        assertValidationErrorField(response, "validateRequestBody.param.secondCode", "Second second must match pattern");
+        ValidationAssertions.assertValidationErrorTitle(response);
+        ValidationAssertions.assertValidationErrorDetails(response);
+        ValidationAssertions.assertValidationErrorStatus(response, HttpStatus.SC_BAD_REQUEST);
```
Comment (pjgg) [inline http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnRequestBodyRouteHandlerIT.java:51]: "ValidationAssertions" -> import...

File: http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnRequestParamRouteHandlerIT.java [hunk 7]
```diff
@@ -45,11 +43,13 @@ public void shouldGetValidationErrorsWhenAllParamsAreWrong() {
                 .then()
                 .statusCode(HttpStatus.SC_BAD_REQUEST)
                 .extract().as(ValidationErrorResponse.class);
-        assertValidationErrorTitle(response);
-        assertValidationErrorDetails(response);
-        assertValidationErrorStatus(response, HttpStatus.SC_BAD_REQUEST);
-        assertValidationErrorField(response, "validateRequestMultipleParam.firstParam", "First param must have 3 characters");
-        assertValidationErrorField(response, "validateRequestMultipleParam.secondParam", "Second param must match pattern");
+        ValidationAssertions.assertValidationErrorTitle(response);
```
Comment (pjgg) [inline http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnRequestParamRouteHandlerIT.java:46]: "ValidationAssertions" -> import

File: http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnResponseRouteHandlerIT.java [hunk 8]
```diff
@@ -33,10 +30,10 @@ public void shouldGetValidationErrorWhenUniResponseIdIsWrong() {
                 .statusCode(HttpStatus.SC_INTERNAL_SERVER_ERROR)
                 .extract().as(ValidationErrorResponse.class);
 
-        assertValidationErrorTitle(response);
-        assertValidationErrorDetails(response);
-        assertValidationErrorStatus(response, HttpStatus.SC_INTERNAL_SERVER_ERROR);
-        assertValidationErrorField(response, "id", "id can't be null");
+        ValidationAssertions.assertValidationErrorTitle(response);
```
Comment (pjgg) [inline http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnResponseRouteHandlerIT.java:33]: "ValidationAssertions" -> import

File: javaee-like-getting-started/src/test/java/io/quarkus/ts/javaee/gettingstarted/FallbackResourceIT.java [hunk 9]
```diff
@@ -25,7 +25,7 @@ public void shouldReturnWorked() {
                 .when().get("/fallback/work")
                 .then()
                 .statusCode(200)
-                .body(is(FallbackResource.WORKED));
+                .body(CoreMatchers.is(FallbackResource.WORKED));
```
Comment (pjgg) [inline javaee-like-getting-started/src/test/java/io/quarkus/ts/javaee/gettingstarted/FallbackResourceIT.java:28]: "static class" should be defined as an import

**Review by pjgg — CHANGES_REQUESTED** [review]
Cosmetic changes

File: http/graphql-telemetry/src/test/java/io/quarkus/ts/http/graphql/telemetry/TelemetryIT.java (hunk 3 above)
Comment (michalvavrik) [reply http/graphql-telemetry/src/test/java/io/quarkus/ts/http/graphql/telemetry/TelemetryIT.java]: True!

**Comment by michalvavrik** [comment]
@pjgg you're absolutely right, I reverted changes unnecessary changes. The feedback has been very much appreciated.

**Review by pjgg — APPROVED** [review]
LGTM

**Comment by michalvavrik** [comment]
Fixed typo, thnx

**Comment by pjgg** [comment]
Kafka errors are not related to this PR.

**Comment by rsvoboda** [comment]
@michalvavrik just fyi:`ChuckNorrisResourceIT` changes for span name were trigger by https://github.com/quarkusio/quarkus/pull/24017

File: http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/ChuckNorrisResourceIT.java [hunk 10]
```diff
@@ -101,7 +101,7 @@ public void getTimeoutWhenResponseItsTooSlow() {
     @Test
     public void endpointShouldTrace() {
         final int pageLimit = 50;
-        final String expectedOperationName = "trace/ping";
+        final String expectedOperationName = "/trace/ping";
```
Comment (michalvavrik) [inline http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/ChuckNorrisResourceIT.java:104]: @pjgg this line can't be backported for only 2.8.0.CR1 + is affected.

**Pull request merged**